docs/set_up_transfer_and_export_device.rst

```diff
@@ -0,0 +1,230 @@
+.. |screenshot of Disk Utility application| image:: images/screenshots/disk-utility.png
+.. |screenshot of the menu to create a new partition in the Disk Utility application| image:: images/screenshots/create-partition.png
+.. |screenshot of passphrase selection prompt in the Disk Utility application| image:: images/screenshots/create-passphrase.png
+
+.. [#] Tails screenshots were taken on Tails 3.0.1. Please make an issue on
+       GitHub if you are using the most recent version of Tails and the
+       interface is different from what you see here.
+
+Create a USB *Export Device*
+----------------------------
+We recommend using a fully encrypted USB drive for copying files off the
+*Secure Viewing Station*. This is even more important than for the
+*Transfer Device*, as the risk of accidentally leaving decrypted files on the
+*Export Device* is significant.
+
+Because the *Export Device* will need to be mounted on both Tails and the
+journalist's everyday workstation, you will need to use an encryption scheme
+that works on both operating systems.
+
+This is the case for many hardware-encrypted USB drives. When considering
+a hardware solution, we recommend selecting a vendor that has fully opened
+the source code and specifications of their devices and encouraged third party
+audits.
+
+VeraCrypt is a good alternative to hardware-based encryption. It is actively
+maintained cross-platform software that has been independently audited and is
+free to use.
+
+VeraCrypt-encrypted media can be opened in the Tails operating system and on
+common Linux distributions without installing additional software. To open
+VeraCrypt media on Windows or Mac workstations, or to create VeraCrypt drives,
+you need to install the VeraCrypt software. The `guide by Freedom of the Press
+Foundation <https://freedom.press/training/encryption-toolkit-media-makers-veracrypt-guide/>`__
+provides instructions for encrypting storage media using VeraCrypt.
+
+Keep in mind that each journalist using a Windows or Mac workstation will need
+to have the VeraCrypt software installed on their computer to access the encrypted
+*Export Device*.
+
+.. tip::
+
+   Larger organizations may want to consider setting up a controlled environment
+   for creating VeraCrypt-encrypted *Export Devices* and providing them to
+   journalists, to ensure that each drive is provisioned in a secure manner.
+
+As with the *Transfer Device*, we recommend storing the passphrase in the
+password manager of each user who will use a given *Export Device*.
+
+Limiting write access
+~~~~~~~~~~~~~~~~~~~~~
+If you re-use the same *Export Device* for multiple copy operations, there is
+the risk of introducing malware to the *Secure Viewing Station* from your
+network. Depending on your threat model, there are steps you may want to take to
+mitigate that risk.
+
+One option is to restrict write access to the *Export Device* before it is
```

Comment:
As a recommendation, this seems a bit unbalanced. To derive benefit from it, an organization using digital forensics tools would need to also have a dedicated workstation for working with files, strong protocols around everything they're doing, and would need to be applying those to *all* submissions that they copy from the SVS. Outside of this context, it seems unrealistic and creates a barrier to entry for organizations that are not as up-to-speed and want to do SecureDrop the "right" way.

Reply:
Per discussion today, prioritized the low-cost option of a USB with a physical switch in 60cc802531e97019bdc76c1723a0dde6eff66494.

Reply:
Another area where write protection would be useful is also for the transfer device (if it is not wiped after each transfer): Enable write protection when connecting to the  SVS to avoid data exfiltration data via the transfer device to the journalist workstation. 

Perhaps prefacing the section with "something to consider" or adding (optional) to the title of the section, making it clear that it's an optional recommendation, which we can strengthen once we've used/practiced these a bit more.

Reply:
It strikes me as a lower risk scenario (because both machines are on Tails, and because the Transfer Device is only ever meant to hold `.gpg` files), but will poke at the language a bit.

Reply:
looks good.